hyper/services/config/config.go

```diff
@@ -34,11 +37,18 @@ type EC2RemoteConfiguration struct {
 	AccessKey string `mapstructure:"access_key" json:"access_key"`
 	Secret    string `mapstructure:"secret" json:"secret"`
 	Region    string `mapstructure:"region" json:"region"`
+	Token     string
 }
 type Configuration struct {
 	SchemaVersion string                         `mapstructure:"schema_version" json:"schema_version"`
 	Remotes       map[string]RemoteConfiguration `mapstructure:"remotes" json:"remotes"`
 }
+type NamedProfileConfiguration struct {
```

Comment:
We've got a folder/package for types now that we will want to put this in. It cleans up the files and allows us to reference them without worrying about creating circular imports